operator/src/main/kubernetes/kubernetes.yml

```diff
@@ -129,4 +133,3 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: keycloak-operator
-    namespace: keycloak
```

Comment:
Just to understand what happens when removing the namespace: Will it then use the namespace associated with the default namespace this YAML is deployed to?

Reply:
I believe the current https://www.keycloak.org/operator/installation are broken then. The instructions show how to install the Operator in the default namespace, but they don't specify that this namespace field needs to be updated, therefore the ServiceAccount will not get the expected permissions as the `ClusterRoleBinding` subject is defined in the `keycloak` namespace.

I suggest that we change the namespace of the `ClusterRoleBinding` subject to `default` and update the docs to state that the namespace value must be changed if deploying to a non-default namespace. I can also include an example patch command showing how to do this.

@shawkins  WDYT?

Reply:
@ryanemerson are you referring to https://www.keycloak.org/operator/installation#_installing_by_using_kubectl_without_operator_lifecycle_manager - that is correct, I don't believe that was updated to reflect the README.

> I suggest that we change the namespace of the ClusterRoleBinding subject to default and update the docs to state that the namespace value must be changed if deploying to a non-default namespace. I can also include an example patch command showing how to do this.

You'd at least have to update the README, and possibly other places that are assuming the keycloak namespace by default.

Reply:
Alternatively we update the guide to always install to the `keycloak` namespace and let users know they must patch the `ClusterRoleBinding` if they want to use a different namespace?

Reply:
The kustomize files aren't available at https://github.com/keycloak/keycloak-k8s-resources, so I have updated the guide to use the patch command:

```
kubectl patch clusterrolebinding keycloak-operator-clusterrole-binding --type='json' -p='[{"op": "replace", "path": "/subjects/0/namespace", "value":"custom-namespace"}]'
```